hack/build.sh

```diff
@@ -80,16 +80,17 @@ run() {
     exit 0
   fi
 
+  # Cross compile only
+  if $(has_flag --all -x); then
+    cross_build || (echo "✋ Cross platform build failed" && exit 1)
+    exit 0
+  fi
+
   # Default flow
   codegen
```

Comment:
Do you know why codegen reaches out to the internet ?

Reply:
I think this is because we do `update_deps` in codegen among other steps, and `update_deps` invokes `go mod tidy` and `go mod vendor`, will need to understand what `go mod *` performs underneath.